server/cmd/mattermost/commands/db.go

```diff
@@ -124,11 +122,11 @@ func initDbCmdF(command *cobra.Command, _ []string) error {
 }
 
 func resetCmdF(command *cobra.Command, args []string) error {
-	a, err := InitDBCommandContextCobra(command, app.SkipPostInitialization())
+	cfgDSN := getConfigDSN(command, config.GetEnvironment())
```

Comment:
Not blocking the PR with this, but if we're going to follow this same pattern for other commands, I think we could refactor this into a helper similar to the `InitDBCommandContextCobra` that returns a store initialized from the config.

Reply:
In fact, we can just use command for that similar to `InitDBCommandContextCobra`, added the suggestion.